### ISSUE TITLE
WCAG 1.1.1: Ensures <img> elements have alternate text or a role of none or presentation (img)

### ISSUE THREAD

**Issue by vkv20** [vkv20/Accesspro#1]
#### Issue

Title: WCAG 4.1.2: Ensures buttons have discernible text (chatgpt-sidebar,.w-\[--btn-size\])
Tags: Accessibility, WCAG 4.1.2, button-name

Issue: Ensures buttons have discernible text (button-name - https://accessibilityinsights.io/info-examples/web/button-name)

Target application: Websites To Practice Testing | Ministry of Testing - https://www.ministryoftesting.com/articles/websites-to-practice-testing

Element path: chatgpt-sidebar,.w-\[--btn-size\]

Snippet: <button class="w-[--btn-size] h-[--btn-size] flex justify-center items-center cursor-pointer p-0 rounded-full border-none outline-none bg-transparent hover:bg-primary4 text-black-text2 hover:text-primary5 transition-colors">

How to fix: 
Fix any of the following:
  Element does not have inner text that is visible to screen readers
  aria-label attribute does not exist or is empty
  aria-labelledby attribute does not exist, references elements that do not exist or references elements that are empty
  Element has no title attribute
  Element's default semantics were not overridden with role="none" or role="presentation"

Environment: Chrome version 128.0.0.0

====

This accessibility issue was found using Accessibility Insights for Web 2.44.0 (axe-core 4.9.1), a tool that helps find and fix accessibility issues. Get more information & download this tool at http://aka.ms/AccessibilityInsights.

`Ensures <img> elements have alternate text or a role of none or presentation` ([image-alt](https://accessibilityinsights.io/info-examples/web/image-alt))
#### Target application
[Websites To Practice Testing | Ministry of Testing](https://www.ministryoftesting.com/articles/websites-to-practice-testing)
#### Element path
.mb-3.align-items-center.flex-column:nth-child(1) > .card.shadow > .resource-detail.card-body > .details.gap-3 > .image > img
#### Snippet
`
